### DISCUSSION TITLE
Can't Get Co-Pilot working with VS code.

### DISCUSSION THREAD

**Discussion by onticinc** [community/community#46567] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have tried multiple times to get co-pilot working with my computer and would love to give it a try. 

It is installed, and I have verified my github account but it will not show me suggestions. 

What am I overlooking or what do I need to do in order to make it work? 

Thanks for your help.

**Comment by SilasRodrigues19** [comment]
Hey @onticinc 

Check if it is active on your account.

https://github.com/github-copilot/signup

Then install the [extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) in VSCode and login with your GitHub account

**Reply by onticinc** [reply]
Hi Silas, 

Thanks for getting back to me. I currently have the 60day trial period active on my account and have installed the extension and logged in. 

I tried walking through the debugging steps that were in the docs but couldn't figure it out. Do you have any further suggestions? 

Thanks
Mike

**Reply by SilasRodrigues19** [reply]
Reinstall the extension and login again, then check if you are giving any errors.

**Comment by onticinc** [comment]
I have tried that three times now, I even tried going back to an older version. I also uninstalled and reinstalled it with and without my vpn turned on. Do I need to reboot my computer after it is installed? 

Is there a way to change any settings or that might apply with VS code or check that it is connecting with github? 

I am currently working on a Next JS app that has both typescript and javascript files in the project. I have tried it on both types of file extensions.

**Comment by davecheney** [comment]
@onticinc I'm sorry you are having difficulties. Is it possible that your internet connection is being filtered by your ISP, employer, or government?

**Reply by onticinc** [reply]
@davecheney I don't think its being filtered, sparklight is my ISP and I am paying for it... ChatGPT works fine, I don't know if that is coming from a similar provider as co-pilot.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by onticinc** [reply]
When I go to View -> Output nothing shows up. 

If I use the command palette and hit Shift+Command+P and search for Copilot Collect Diagnostics, this is what comes back. 

## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Extensions

- Copilot Version: 1.71.8269
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Darwin
- Operating system version: 21.6.0
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 -

**Comment by jge162** [comment]
Hello @onticinc I’ve see some issues in posts about copilot. The devs said to use copilot nightly in the meantime.[Here](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly)

**Comment by kimadeline** [comment]
Hi @onticinc, 

Could you try copying here the content of the developer console (Command Palette > "Developer: Toggle Developer Tools")? Do you also have the possibility to try installing Copilot on a different editor (IntelliJ, Visual Studio or Neovim) and see if it works there? If it does, that's an issue with the VS Code extension, and if it doesn't work, that would be something with your setup or your account.

Thank you!

**Comment by onticinc** [comment, marked as answer]
Figured it out... it was this setting. 

"editor.inlineSuggest.enabled": false,

Didn't notice that at first. Thanks for your help.
